docs-2.0/3.ngql-guide/5.operators/7.string.md

```diff
@@ -138,3 +139,13 @@ nebula> YIELD 'apple' ENDS WITH 'b';
 | false               |
 +---------------------+
 ```
+
+* Regular expressions
+
+Nebula Graph supports filtering by using regular expressions. The regular expression syntax is inherited from `st::regex`. You can match on regular expressions by using `=~ 'regexp'`. For example:
+
+```ngql
+nebula> FETCH PROP ON player WHERE player.name =~ 'Tony.*' YIELD player.name;
+```
+
+The preceding statement returns players whose name starts with 'Tony'.
```

Comment:
players whose name starts with 'Tony'. ==> the players whose names start with 'Tony'.